### PR TITLE
Closes #11944: changes the way the system notifications are tested

### LINKED ISSUE #11944: FNX2-13683 ⁃ Intermittent UI test failure - testDownloadNotification - API28
### Firebase Test Run:
Copied from https://github.com/mozilla-mobile/fenix/issues/5462#issuecomment-561760268
intermittent fail on API 28
https://console.firebase.google.com/u/1/project/moz-fenix/testlab/histories/bh.66b7091e15d53d45/matrices/8780279402119153460/executions/bs.e7261e4c00726c9e/testcases/1/test-cases

"In the video, the notification shade has a ton of other things and the notification we want to check against must be off-screen. Test fix would see if we can scroll the notification into-view."

This test was disabled a while ago, I opened this issue to track it and re-enable when possible.

### PR DESCRIPTION
**Pull request by sv-ohorvath**
Firebase tests runs on API 28: https://github.com/mozilla-mobile/fenix/runs/1008390539 (which used to fail before this)
Tests passed locally on API 27.

### CODE REVIEW

**Comment by sv-ohorvath** [comment]
downloads fail on Nightly: https://github.com/mozilla-mobile/fenix/issues/14052 - waiting for this to be fixed before reopening the PR

**Comment by sv-ohorvath** [comment]
@AaronMT the download problem was fixed and this is ready for review again when you can. Thank you!

File: app/src/androidTest/java/org/mozilla/fenix/ui/robots/NotificationRobot.kt [hunk 1]
```diff
@@ -17,23 +18,20 @@ class NotificationRobot {
...
-                    UiSelector().text(notificationMessage), notificationMessage,
-                    true
-                )
-                notification.exists()
-                notificationFound = true
-            } catch (e: Resources.NotFoundException) {
-                e.printStackTrace()
-            }
+        if (!notificationFound) {
+            notificationTray().swipeUp(2)
```
Comment (AaronMT) [inline app/src/androidTest/java/org/mozilla/fenix/ui/robots/NotificationRobot.kt:32]: What is the 2? I would add a comment.

Comment (sv-ohorvath) [reply app/src/androidTest/java/org/mozilla/fenix/ui/robots/NotificationRobot.kt:33]: done

**Pull request merged**